.github/workflows/clp-uv-checks.yaml

```
@@ -0,0 +1,50 @@
+
+jobs:
+  rust-checks:
+    strategy:
+      matrix:
+        os:
+          - "macos-15"
+          - "ubuntu-22.04"
+          - "ubuntu-24.04"
+    runs-on: "${{matrix.os}}"
+    steps:
+      - # v5.0.0
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)